src/sentry/api/serializers/models/organization.py

```diff
@@ -180,19 +208,23 @@ def serialize(self, obj, attrs, user):
         }
 
 
-class OnboardingTasksSerializer(Serializer):
-    def get_attrs(self, item_list, user, **kwargs):
+class OnboardingTasksSerializer(Serializer):  # type: ignore
+    def get_attrs(
+        self, item_list: OrganizationOnboardingTask, user: User, **kwargs: Any
+    ) -> MutableMapping[OrganizationOnboardingTask, MutableMapping[str, Any]]:
         # Unique user list
         users = {item.user for item in item_list if item.user}
         serialized_users = serialize(users, user, UserSerializer())
         user_map = {user["id"]: user for user in serialized_users}
 
-        data = {}
+        data: MutableMapping[OrganizationOnboardingTask, MutableMapping[str, Any]] = {}
```

Comment:
Is this really `MutableMapping[str, Any]`? It seems like `serialized_users` should be a sequence of a concrete type. Is that because that serializer isn't typed yet?

Reply:
these types could be more detailed but I judged that the extra detail is more annoying than helpful in this case.

Reply:
Why do you think it's more annoying?

Reply:
this type would only be used in the `serialize` function below, and given that `OnboardingTasks` is mainly data that would be used internally (people using the public API probably aren't interested in this field), the only value of typing it is to make the code more documented/readable, and in this case, it was quite clear from before what this function was returning.

If we look at all other serializers that I haven't touched for these public API doc changes, all of the functions return ambiguous types, so I believed I was simply following the previous standard that was set. 

At any rate, I'll go ahead and push up a change to type the value here.

Let me know if that makes sense, or should we try and go back and fix all of the ambiguous types that we've already added in the serializers?

I am attempting to move somewhat fast here as this is primarily product feature work and not a tech debt / code doc effort, so going forward I will only type the parts that I need for documenting the public API and leave the files off `mypy.ini`.